### DISCUSSION TITLE
Stopped working

### DISCUSSION THREAD

**Discussion by kleuter** [plausible/analytics#4859] [Self-Hosted Support]
Hi,

I have migrated to v2.1 without any problems earlier this year and today found out that it's not working.
Nginx configured with a reverse proxy gives has 502 Bad Gateway.

Any ideas please? Config files are in the end. Thanks.

Docker log:

```
 ✔ Network plausible_default                  Created                                                                                                       0.1s
 ✔ Container plausible-plausible_events_db-1  Created                                                                                                       0.1s
 ✔ Container plausible-mail-1                 Created                                                                                                       0.1s
 ✔ Container plausible-plausible_db-1         Created                                                                                                       0.1s
 ✔ Container plausible-plausible-1            Created                                                                                                       0.0s
Attaching to mail-1, plausible-1, plausible_db-1, plausible_events_db-1
plausible_db-1         |
plausible_db-1         | PostgreSQL Database directory appears to contain a database; Skipping initialization
plausible_db-1         |
plausible_db-1         | 2024-11-26 15:55:39.531 UTC [1] LOG:  starting PostgreSQL 14.8 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r10)                                                                                        12.2.1 20220924, 64-bit
plausible_db-1         | 2024-11-26 15:55:39.534 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
plausible_db-1         | 2024-11-26 15:55:39.534 UTC [1] LOG:  listening on IPv6 address "::", port 5432
plausible_db-1         | 2024-11-26 15:55:39.540 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
plausible_db-1         | 2024-11-26 15:55:39.549 UTC [21] LOG:  database system was shut down at 2024-11-26 15:43:55 UTC
plausible_db-1         | 2024-11-26 15:55:39.587 UTC [1] LOG:  database system is ready to accept connections
mail-1                 |     1 LOG: MAIN
mail-1                 |     1   exim 4.89 daemon started: pid=1, -q30m, listening for SMTP on port 25 (IPv6 and IPv4)
plausible_events_db-1  | ClickHouse Database directory appears to contain a database; Skipping initialization
plausible_events_db-1  | Processing configuration file '/etc/clickhouse-server/config.xml'.
plausible_events_db-1  | Merging configuration file '/etc/clickhouse-server/config.d/docker_related_config.xml'.
plausible_events_db-1  | Merging configuration file '/etc/clickhouse-server/config.d/logging.xml'.
plausible_events_db-1  | Logging warning to /var/log/clickhouse-server/clickhouse-server.log
plausible_events_db-1  | Logging errors to /var/log/clickhouse-server/clickhouse-server.err.log
plausible_events_db-1  | 2024.11.26 15:55:40.424192 [ 1 ] {} <Warning> Context: Linux threads max count is too low. Check /proc/sys/kernel/threads-max
plausible_events_db-1  | 2024.11.26 15:55:40.440485 [ 1 ] {} <Warning> Context: Available memory at server startup is too low (2GiB).
plausible_events_db-1  | 2024.11.26 15:55:41.420868 [ 1 ] {} <Warning> Application: Listen [::]:9009 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error                                                                                       : EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4,                                                                                        then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0                                                                                       .0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 15:55:42.315164 [ 1 ] {} <Warning> Application: Listen [::]:8123 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error                                                                                       : EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4,                                                                                        then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0                                                                                       .0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 15:55:42.318349 [ 1 ] {} <Warning> Application: Listen [::]:9000 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error                                                                                       : EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4,                                                                                        then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0                                                                                       .0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 15:55:42.319533 [ 1 ] {} <Warning> Application: Listen [::]:9004 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error                                                                                       : EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4,                                                                                        then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0                                                                                       .0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 15:55:42.320336 [ 1 ] {} <Warning> Application: Listen [::]:9005 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error                                                                                       : EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4,                                                                                        then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0                                                                                       .0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible-1            | Loading plausible..
plausible-1            | Starting dependencies..
plausible-1            | Starting repos..
plausible-1            | create Plausible.Repo database if it doesn't exist
plausible-1            | create Plausible.IngestRepo database if it doesn't exist
plausible_events_db-1  | 2024.11.26 15:55:51.584359 [ 48 ] {9d2b4ac9-ce6b-4535-9944-dbcb79aebf00} <Error> executeQuery: Code: 82. DB::Exception: Database plausible_events_db already exists. (DATABASE_ALREADY_EXISTS) (version 24.3.3.102 (official build)) (from 172.18.0.5:48598) (in query: CREATE DATABASE "plausible_events_db"), Stack trace (when copying this message, always include the lines below):
plausible_events_db-1  |
plausible_events_db-1  | 0. DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x000000000cbc851b
plausible_events_db-1  | 1. DB::Exception::Exception<String&>(int, FormatStringHelperImpl<std::type_identity<String&>::type>, String&) @ 0x000000000766ce63
plausible_events_db-1  | 2. DB::InterpreterCreateQuery::createDatabase(DB::ASTCreateQuery&) @ 0x0000000010b5e982
plausible_events_db-1  | 3. DB::InterpreterCreateQuery::execute() @ 0x0000000010b812be
plausible_events_db-1  | 4. DB::executeQueryImpl(char const*, char const*, std::shared_ptr<DB::Context>, DB::QueryFlags, DB::QueryProcessingStage::Enum, DB::ReadBuffer*) @ 0x000000001126e162
plausible_events_db-1  | 5. DB::executeQuery(DB::ReadBuffer&, DB::WriteBuffer&, bool, std::shared_ptr<DB::Context>, std::function<void (DB::QueryResultDetails const&)>, DB::QueryFlags, std::optional<DB::FormatSettings> const&, std::function<void (DB::IOutputFormat&)>) @ 0x000000001127341e
plausible_events_db-1  | 6. DB::HTTPHandler::processQuery(DB::HTTPServerRequest&, DB::HTMLForm&, DB::HTTPServerResponse&, DB::HTTPHandler::Output&, std::optional<DB::CurrentThread::QueryScope>&, StrongTypedef<unsigned long, ProfileEvents::EventTag> const&) @ 0x00000000122f0075
plausible_events_db-1  | 7. DB::HTTPHandler::handleRequest(DB::HTTPServerRequest&, DB::HTTPServerResponse&, StrongTypedef<unsigned long, ProfileEvents::EventTag> const&) @ 0x00000000122f4b34
plausible_events_db-1  | 8. DB::HTTPServerConnection::run() @ 0x00000000123740da
plausible_events_db-1  | 9. Poco::Net::TCPServerConnection::start() @ 0x0000000014c9b8d2
plausible_events_db-1  | 10. Poco::Net::TCPServerDispatcher::run() @ 0x0000000014c9c719
plausible_events_db-1  | 11. Poco::PooledThread::run() @ 0x0000000014d94e81
plausible_events_db-1  | 12. Poco::ThreadImpl::runnableEntry(void*) @ 0x0000000014d9341d
plausible_events_db-1  | 13. ? @ 0x00007ff17af97609
plausible_events_db-1  | 14. ? @ 0x00007ff17aebc353
plausible_events_db-1  |
plausible_events_db-1  | 2024.11.26 15:55:51.586795 [ 48 ] {9d2b4ac9-ce6b-4535-9944-dbcb79aebf00} <Error> DynamicQueryHandler: Code: 82. DB::Exception: Database plausible_events_db already exists. (DATABASE_ALREADY_EXISTS), Stack trace (when copying this message, always include the lines below):
plausible_events_db-1  |
plausible_events_db-1  | 0. DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x000000000cbc851b
plausible_events_db-1  | 1. DB::Exception::Exception<String&>(int, FormatStringHelperImpl<std::type_identity<String&>::type>, String&) @ 0x000000000766ce63
plausible_events_db-1  | 2. DB::InterpreterCreateQuery::createDatabase(DB::ASTCreateQuery&) @ 0x0000000010b5e982
plausible_events_db-1  | 3. DB::InterpreterCreateQuery::execute() @ 0x0000000010b812be
plausible_events_db-1  | 4. DB::executeQueryImpl(char const*, char const*, std::shared_ptr<DB::Context>, DB::QueryFlags, DB::QueryProcessingStage::Enum, DB::ReadBuffer*) @ 0x000000001126e162
plausible_events_db-1  | 5. DB::executeQuery(DB::ReadBuffer&, DB::WriteBuffer&, bool, std::shared_ptr<DB::Context>, std::function<void (DB::QueryResultDetails const&)>, DB::QueryFlags, std::optional<DB::FormatSettings> const&, std::function<void (DB::IOutputFormat&)>) @ 0x000000001127341e
plausible_events_db-1  | 6. DB::HTTPHandler::processQuery(DB::HTTPServerRequest&, DB::HTMLForm&, DB::HTTPServerResponse&, DB::HTTPHandler::Output&, std::optional<DB::CurrentThread::QueryScope>&, StrongTypedef<unsigned long, ProfileEvents::EventTag> const&) @ 0x00000000122f0075
plausible_events_db-1  | 7. DB::HTTPHandler::handleRequest(DB::HTTPServerRequest&, DB::HTTPServerResponse&, StrongTypedef<unsigned long, ProfileEvents::EventTag> const&) @ 0x00000000122f4b34
plausible_events_db-1  | 8. DB::HTTPServerConnection::run() @ 0x00000000123740da
plausible_events_db-1  | 9. Poco::Net::TCPServerConnection::start() @ 0x0000000014c9b8d2
plausible_events_db-1  | 10. Poco::Net::TCPServerDispatcher::run() @ 0x0000000014c9c719
plausible_events_db-1  | 11. Poco::PooledThread::run() @ 0x0000000014d94e81
plausible_events_db-1  | 12. Poco::ThreadImpl::runnableEntry(void*) @ 0x0000000014d9341d
plausible_events_db-1  | 13. ? @ 0x00007ff17af97609
plausible_events_db-1  | 14. ? @ 0x00007ff17aebc353
plausible_events_db-1  |  (version 24.3.3.102 (official build))
plausible-1            | Creation of Db successful!
plausible-1            | Loading plausible..
plausible-1            | Starting dependencies..
plausible-1            | Starting repos..
plausible-1            | Running migrations for Elixir.Plausible.Repo
plausible-1            | Running migrations for Elixir.Plausible.IngestRepo
plausible-1            | Migrations successful!
plausible-1            | 15:56:05.139 [error] Failed to send e-mail
plausible-1            | 15:56:06.217 [error] GenServer :tzdata_release_updater terminating
plausible-1            | ** (ArgumentError) errors were found at the given arguments:
plausible-1            |
plausible-1            |   * 2nd argument: not a tuple
plausible-1            |
plausible-1            |     :erlang.element(1, :error)
plausible-1            |     (tzdata 1.1.1) lib/tzdata/util.ex:223: Tzdata.Util.to_int/1
plausible-1            |     (tzdata 1.1.1) lib/tzdata/parser.ex:38: Tzdata.Parser.process_rule/1
plausible-1            |     (tzdata 1.1.1) lib/tzdata/parser.ex:24: Tzdata.Parser.process_tz_list/1
plausible-1            |     (tzdata 1.1.1) lib/tzdata/parser.ex:86: Tzdata.Parser.process_zone/5
plausible-1            |     (tzdata 1.1.1) lib/tzdata/parser.ex:24: Tzdata.Parser.process_tz_list/1
plausible-1            |     (tzdata 1.1.1) lib/tzdata/parser.ex:86: Tzdata.Parser.process_zone/5
plausible-1            |     (tzdata 1.1.1) lib/tzdata/parser.ex:24: Tzdata.Parser.process_tz_list/1
plausible-1            | Last message: :check_if_time_to_update
^CGracefully stopping... (press Ctrl+C again to force)
[+] Stopping 4/4
 ✔ Container plausible-plausible-1            Stopped                                                                                                                                                                                              3.1s
 ✔ Container plausible-plausible_db-1         Stopped                                                                                                                                                                                              0.5s
 ✔ Container plausible-plausible_events_db-1  Stopped                                                                                                                                                                                              1.9s
 ✔ Container plausible-mail-1                 Stopped                                                                                                                                                                                             10.6s
canceled
xpisik@webstatistics:~/plausible$ ls
clickhouse  docker-compose.yml  geoip  kubernetes  mail.log  plausible-conf.env  plausible-conf.env.save  README.md  reverse-proxy
xpisik@webstatistics:~/plausible$ sudo nano plausible-conf.env
xpisik@webstatistics:~/plausible$ sudn nano docker-compose.yml

Command 'sudn' not found, did you mean:

  command 'sudo' from deb sudo (1.8.31-1ubuntu1.5)
  command 'sudo' from deb sudo-ldap (1.8.31-1ubuntu1.5)

Try: sudo apt install <deb name>

xpisik@webstatistics:~/plausible$ sudo nano docker-compose.yml
xpisik@webstatistics:~/plausible$ sudo docker compose up > docker.log
[+] Running 4/0
 ✔ Container plausible-mail-1                 Created0.0s
 ✔ Container plausible-plausible_db-1         Created0.0s
 ✔ Container plausible-plausible_events_db-1  Created0.0s
 ✔ Container plausible-plausible-1            Created0.0s
plausible_db-1         | 2024-11-26 16:03:02.915 UTC [1] LOG:  starting PostgreSQL 14.8 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r10) 12.2.1 20220924, 64-bit
plausible_db-1         | 2024-11-26 16:03:02.917 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
plausible_db-1         | 2024-11-26 16:03:02.918 UTC [1] LOG:  listening on IPv6 address "::", port 5432
plausible_db-1         | 2024-11-26 16:03:02.921 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
plausible_db-1         | 2024-11-26 16:03:02.934 UTC [22] LOG:  database system was shut down at 2024-11-26 15:58:06 UTC
plausible_db-1         | 2024-11-26 16:03:02.953 UTC [1] LOG:  database system is ready to accept connections
mail-1                 |     1 LOG: MAIN
mail-1                 |     1   exim 4.89 daemon started: pid=1, -q30m, listening for SMTP on port 25 (IPv6 and IPv4)
plausible_events_db-1  | Processing configuration file '/etc/clickhouse-server/config.xml'.
plausible_events_db-1  | Merging configuration file '/etc/clickhouse-server/config.d/docker_related_config.xml'.
plausible_events_db-1  | Merging configuration file '/etc/clickhouse-server/config.d/logging.xml'.
plausible_events_db-1  | Logging warning to /var/log/clickhouse-server/clickhouse-server.log
plausible_events_db-1  | Logging errors to /var/log/clickhouse-server/clickhouse-server.err.log
plausible_events_db-1  | 2024.11.26 16:03:03.837102 [ 1 ] {} <Warning> Context: Linux threads max count is too low. Check /proc/sys/kernel/threads-max
plausible_events_db-1  | 2024.11.26 16:03:03.840829 [ 1 ] {} <Warning> Context: Available memory at server startup is too low (2GiB).
plausible_events_db-1  | 2024.11.26 16:03:04.748897 [ 1 ] {} <Warning> Application: Listen [::]:9009 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 16:03:05.906323 [ 1 ] {} <Warning> Application: Listen [::]:8123 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 16:03:05.909591 [ 1 ] {} <Warning> Application: Listen [::]:9000 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 16:03:05.910375 [ 1 ] {} <Warning> Application: Listen [::]:9004 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 16:03:05.911318 [ 1 ] {} <Warning> Application: Listen [::]:9005 failed: Poco::Exception. Code: 1000, e.code() = 0, DNS error: EAI: Address family for hostname not supported (version 24.3.3.102 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>
plausible_events_db-1  | 2024.11.26 16:03:15.291502 [ 49 ] {6b40d27b-1d1b-4b82-9d71-35b718b8a3a2} <Error> executeQuery: Code: 82. DB::Exception: Database plausible_events_db already exists. (DATABASE_ALREADY_EXISTS) (version 24.3.3.102 (official build)) (from 172.18.0.5:51692) (in query: CREATE DATABASE "plausible_events_db"), Stack trace (when copying this message, always include the lines below):
plausible_events_db-1  |
plausible_events_db-1  | 0. DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x000000000cbc851b
plausible_events_db-1  | 1. DB::Exception::Exception<String&>(int, FormatStringHelperImpl<std::type_identity<String&>::type>, String&) @ 0x000000000766ce63
plausible_events_db-1  | 2. DB::InterpreterCreateQuery::createDatabase(DB::ASTCreateQuery&) @ 0x0000000010b5e982
plausible_events_db-1  | 3. DB::InterpreterCreateQuery::execute() @ 0x0000000010b812be
plausible_events_db-1  | 4. DB::executeQueryImpl(char const*, char const*, std::shared_ptr<DB::Context>, DB::QueryFlags, DB::QueryProcessingStage::Enum, DB::ReadBuffer*) @ 0x000000001126e162
plausible_events_db-1  | 5. DB::executeQuery(DB::ReadBuffer&, DB::WriteBuffer&, bool, std::shared_ptr<DB::Context>, std::function<void (DB::QueryResultDetails const&)>, DB::QueryFlags, std::optional<DB::FormatSettings> const&, std::function<void (DB::IOutputFormat&)>) @ 0x000000001127341e
plausible_events_db-1  | 6. DB::HTTPHandler::processQuery(DB::HTTPServerRequest&, DB::HTMLForm&, DB::HTTPServerResponse&, DB::HTTPHandler::Output&, std::optional<DB::CurrentThread::QueryScope>&, StrongTypedef<unsigned long, ProfileEvents::EventTag> const&) @ 0x00000000122f0075
plausible_events_db-1  | 7. DB::HTTPHandler::handleRequest(DB::HTTPServerRequest&, DB::HTTPServerResponse&, StrongTypedef<unsigned long, ProfileEvents::EventTag> const&) @ 0x00000000122f4b34
plausible_events_db-1  | 8. DB::HTTPServerConnection::run() @ 0x00000000123740da
plausible_events_db-1  | 9. Poco::Net::TCPServerConnection::start() @ 0x0000000014c9b8d2
plausible_events_db-1  | 10. Poco::Net::TCPServerDispatcher::run() @ 0x0000000014c9c719
plausible_events_db-1  | 11. Poco::PooledThread::run() @ 0x0000000014d94e81
plausible_events_db-1  | 12. Poco::ThreadImpl::runnableEntry(void*) @ 0x0000000014d9341d
plausible_events_db-1  | 13. ? @ 0x00007f0907d81609
plausible_events_db-1  | 14. ? @ 0x00007f0907ca6353
plausible_events_db-1  |
plausible_events_db-1  | 2024.11.26 16:03:15.295133 [ 49 ] {6b40d27b-1d1b-4b82-9d71-35b718b8a3a2} <Error> DynamicQueryHandler: Code: 82. DB::Exception: Database plausible_events_db already exists. (DATABASE_ALREADY_EXISTS), Stack trace (when copying this message, always include the lines below):
plausible_events_db-1  |
plausible_events_db-1  | 0. DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x000000000cbc851b
plausible_events_db-1  | 1. DB::Exception::Exception<String&>(int, FormatStringHelperImpl<std::type_identity<String&>::type>, String&) @ 0x000000000766ce63
plausible_events_db-1  | 2. DB::InterpreterCreateQuery::createDatabase(DB::ASTCreateQuery&) @ 0x0000000010b5e982
plausible_events_db-1  | 3. DB::InterpreterCreateQuery::execute() @ 0x0000000010b812be
plausible_events_db-1  | 4. DB::executeQueryImpl(char const*, char const*, std::shared_ptr<DB::Context>, DB::QueryFlags, DB::QueryProcessingStage::Enum, DB::ReadBuffer*) @ 0x000000001126e162
plausible_events_db-1  | 5. DB::executeQuery(DB::ReadBuffer&, DB::WriteBuffer&, bool, std::shared_ptr<DB::Context>, std::function<void (DB::QueryResultDetails const&)>, DB::QueryFlags, std::optional<DB::FormatSettings> const&, std::function<void (DB::IOutputFormat&)>) @ 0x000000001127341e
plausible_events_db-1  | 6. DB::HTTPHandler::processQuery(DB::HTTPServerRequest&, DB::HTMLForm&, DB::HTTPServerResponse&, DB::HTTPHandler::Output&, std::optional<DB::CurrentThread::QueryScope>&, StrongTypedef<unsigned long, ProfileEvents::EventTag> const&) @ 0x00000000122f0075
plausible_events_db-1  | 7. DB::HTTPHandler::handleRequest(DB::HTTPServerRequest&, DB::HTTPServerResponse&, StrongTypedef<unsigned long, ProfileEvents::EventTag> const&) @ 0x00000000122f4b34
plausible_events_db-1  | 8. DB::HTTPServerConnection::run() @ 0x00000000123740da
plausible_events_db-1  | 9. Poco::Net::TCPServerConnection::start() @ 0x0000000014c9b8d2
plausible_events_db-1  | 10. Poco::Net::TCPServerDispatcher::run() @ 0x0000000014c9c719
plausible_events_db-1  | 11. Poco::PooledThread::run() @ 0x0000000014d94e81
plausible_events_db-1  | 12. Poco::ThreadImpl::runnableEntry(void*) @ 0x0000000014d9341d
plausible_events_db-1  | 13. ? @ 0x00007f0907d81609
plausible_events_db-1  | 14. ? @ 0x00007f0907ca6353
plausible_events_db-1  |  (version 24.3.3.102 (official build))
```

[docker-compose.yml.txt](https://github.com/user-attachments/files/17922343/docker-compose.yml.txt)
[nginx.conf.txt](https://github.com/user-attachments/files/17922344/nginx.conf.txt)

**Comment by ruslandoga** [comment]
:wave: @kleuter 

There are no _real_ errors in your logs after you stopped and restarted Docker Compose. Please explain what exactly isn't working as expected or provide full logs for the currently running containers :)

**Comment by kleuter** [comment]
@ruslandoga 

Thanks for the quick follow up. I restarted docker it works now! Not sure what but it happened 20 Nov
![image](https://github.com/user-attachments/assets/c63f3055-3f58-4131-8668-aec2d1831371)

Update: looks like my free space is low, can this be the reason?
![image](https://github.com/user-attachments/assets/f7c674e6-6f35-4b0e-911e-0ee22247ed9d)

![image](https://github.com/user-attachments/assets/a14f18e2-13bb-4bd9-86b0-c79e83183bca)

Update:
Is there a way to optimize plausible databases?

**Reply by ruslandoga** [reply]
> Is there a way to optimize plausible databases?

Yes. I think there have been some discussions about this already, first hit: https://github.com/plausible/analytics/discussions/3722 :)

**tl;dr**

Check `system.parts` table

```console
$ docker compose exec plausible_events_db clickhouse client --database plausible_events_db
```
```sql
:) select table, active, formatReadableSize(sum(bytes_on_disk)) from system.parts group by table, active order by sum(bytes_on_disk) desc;
```

And then maybe drop tables with logs, drop legacy tables (like `events` or `sessions` which were used before v2), drop temporary tables (like `sessions_v2_tmp` which was used in some migration), run `OPTIMIZE ... FINAL` (with care), etc.
